runtime/lua/vim/lsp/snippet/parser.lua

```diff
@@ -1,9 +1,12 @@
 local P = {}
```

Comment:
This module can live in `runtime/lua/vim/lsp/snippet.lua`, don't need a separate `parser.lua`.

Reply:
Alright👍